File: boot-tyb/src/main/java/com/xzm/tyb/vo/AuthVo.java
package com.xzm.tyb.vo;

import io.swagger.annotations.ApiModelProperty;

//@ApiModel("认证Token")
public class AuthVo {
    @ApiModelProperty(value = "认证access_token",required = true)
    private String access_token;

    public String getAccess_token() {
        return access_token;
    }

    public void setAccess_token(String access_token) {
        this.access_token = access_token;
    }
}

File: boot-tyb/src/main/java/com/xzm/tyb/pojo/vo/AuthVo.kt
package com.xzm.tyb.pojo.vo

import io.swagger.annotations.ApiModelProperty

//@ApiModel("认证Token")
open class AuthVo {
    //    @ApiModelProperty(value = "认证access_token",required = true)
    //    private String access_token;
    @ApiModelProperty(value = "认证accessToken", required = false)
    var accessToken: String? = null
}
